src/simpla-admin.html

```diff
@@ -112,8 +135,16 @@
       }
 
       /**
+       * Attach singleton to window on created
+       * @return {[type]} [description]
+       */
+      ready() {
```

Comment:
welp, `ready` seems to work across browsers for me, `attached` fails everywhere. Can readdress later if it becomes an issue